core/src/main/java/io/questdb/griffin/SqlExecutionContextImpl.java

```diff
@@ -62,10 +62,11 @@ public class SqlExecutionContextImpl implements SqlExecutionContext {
     private int jitMode;
     private boolean cloneSymbolTables = false;
 
-    public SqlExecutionContextImpl(CairoEngine cairoEngine, int workerCount) {
+    public SqlExecutionContextImpl(CairoEngine cairoEngine, int workerCount, int sharedWorkerCount) {
         this.cairoConfiguration = cairoEngine.getConfiguration();
         this.workerCount = workerCount;
         assert workerCount > 0;
+        this.sharedWorkerCount = sharedWorkerCount;
```

Comment:
nit: do we want to place `assert sharedWorkerCount > 0` since we have a similar one for `workerCount`?

Reply:
I need to update some tests then